src/server/components/dashboard/listsItems/item/update/actionHandlers/archive.ts

```diff
@@ -13,8 +14,9 @@ export async function archive(req: Request, res: Response): Promise<void> {
     req.flash("successBannerHeading", "Archived");
     req.flash("successBannerColour", "green");
     return res.redirect(listIndexUrl);
-  } catch (error: any) {
-    req.flash("errorMsg", `${listItem.jsonData.organisationName} could not be archived. ${error.message}`);
+  } catch (error: unknown) {
+    req.flash("errorMsg", `${listItem.jsonData.organisationName} could not be archived.`);
+    logger.error((error as Error).message);
```

Comment:
As an fyi, if you stringify an error (whether calling .toString() or template literal), it will return a string containing error name and message, so you don't really need error.message.

Reply:
Ah, that would save me needing to set the type 👍🏿